test/rules/experiments/nnbd/rules/avoid_dynamic_calls.dart

```diff
@@ -120,12 +120,19 @@ void binaryExpressions(dynamic a, int b, bool c) {
 }
 
 void equalityExpressions(dynamic a, dynamic b) {
-  a == b; // LINT
-  a == null; // OK, special cased.
-  a != b; // LINT
+  a == b; // OK, see lint description for details.
+  a == null; // OK.
+  a != b; // OK
   a != null; // OK.
 }
 
+void membersThatExistOnObject(dynamic a) {
+  a.hashCode; // OK
+  a.runtimeType; // OK
+  a.noSuchMethod(null as Invocation); // OK
+  a.toString(); // OK
```

Comment:
Also test tearoffs of `noSuchMethod` and `toString` (OK) and calling these with the wrong number of arguments, or with named arguments (LINT).

Reply:
I did the most I could. I think being more comprehensive than this is likely infeasible, as the analyzer issues compile-time errors already for the wrong number of arguments on these methods, including tear-offs, and I'm not able to lint them as a result.